lib/screens/domain/todos_repository.dart

```diff
@@ -0,0 +1,7 @@
+import 'package:flutter_theme/screens/domain/todos.dart';
+
+abstract class TodosRepository {
+  Future<Todos> addNewTodo(Todos todo);
+  Future<Todos> deleteTodo(Todos todo);
```

Comment:
`Future<void> deleteTodo(Todos todo);`
after delete we don't need return todo because it is no more